header/p2p/server.go

```diff
@@ -163,7 +163,9 @@ func (serv *ExchangeServer) handleRequest(from, to uint64) ([]*header.ExtendedHe
 		return nil, header.ErrHeadersLimitExceeded
 	}
 	log.Debugw("server: handling headers request", "from", from, "to", to)
-	headersByRange, err := serv.store.GetRangeByHeight(serv.ctx, from, to)
+	ctx, cancel := context.WithTimeout(serv.ctx, time.Minute)
+	defer cancel()
```

Comment:
As we discussed onsite, we don't wanna have our peers _subscribe_ for headers. The minute will end up exactly as a subscription, so I  propose to decrease this to 5 seconds. It's enough time to get the data from the disk, and we don't need it for more

Reply:
I'm ok with that